src/nodedef.cpp

```diff
@@ -112,6 +183,64 @@ void NodeBox::deSerialize(std::istream &is)
 		wall_side.MinEdge = readV3F1000(is);
 		wall_side.MaxEdge = readV3F1000(is);
 	}
+	else if (type == NODEBOX_CONNECTED)
```

Comment:
Because you add one more type, please use a switch case on this function to improve condition handling performance.